includes/abstracts/abstract-wc-payment-token.php

```diff
@@ -104,7 +107,7 @@ public function get_type( $context = 'view' ) {
 	 * @return string
 	 */
 	public function get_display_name( $context = 'view' ) {
```

Comment:
Some of the tokens use it when they overwrite `get_display_name` https://github.com/woocommerce/woocommerce/pull/13596/files#diff-ad66f24a4cdcd73abaadc71c054f84a5R42 because it is calling some props. 

I left it because it already existed, however I am having trouble of thinking of a case where you might display this and want the props `get_display_name` uses to be anything other than 'view'.. so I think we can deprecate this.